src/Platforms/OraclePlatform.php

```diff
@@ -596,9 +596,9 @@ private function normalizeIdentifier($name)
      */
     private function addSuffix(string $identifier, string $suffix): string
     {
-        $maxPossibleLengthWithoutSuffix = $this->getMaxIdentifierLength() - strlen($suffix);
-        if (strlen($identifier) > $maxPossibleLengthWithoutSuffix) {
-            $identifier = substr($identifier, 0, $maxPossibleLengthWithoutSuffix);
+        $max = $this->getMaxIdentifierLength() - strlen($suffix);
```

Comment:
Why rename the variable?

Reply:
Simply because the next line got way to long with that variable name

Reply:
Please revert.

Reply:
Done